doc/changelog/03-notations/19149-existing_level_default.rst

```diff
@@ -0,0 +1,6 @@
+- **Changed:**
+  default level of :cmd:`Reserved Notation` to help
+  :ref:`factorize notations <NotationFactorization>`
+  with common prefixes
```

Comment:
Can it be more explicit? E.g. something like: "When no level is specified in :cmd:`Reserved Notation`, the previous notation with maximal common prefix is used, if any, to set a default; this helps to :ref:`factorize notations <NotationFactorization>`".

Reply:
Indeed, done